Dockerfile.j2

```diff
@@ -32,6 +32,7 @@ RUN yum install -y yum-utils https://centos7.iuscommunity.org/ius-release.rpm \
     && rm -rf "$GNUPGHOME" crate-{{ CRATE_VERSION }}.tar.gz.asc \
     && tar -xf crate-{{ CRATE_VERSION }}.tar.gz -C /crate --strip-components=1 \
     && rm crate-{{ CRATE_VERSION }}.tar.gz \
+    && ln -sf /usr/bin/python3.6 /usr/bin/python3
     && ln -sf /usr/bin/python3 /usr/bin/python
```

Comment:
Might want to make the source here `3.6` as well to avoid the other symlink indirection.